docs/guide/headless-cms.md

```diff
@@ -90,10 +96,10 @@ Page REST API requests look very similar to piece requests. They follow the same
 
 ```
 # All pages
-GET https://example.rocks/api/v1/@apostrophecms/page?apikey=000apikey123
```

Comment:
The root query for pages behaves differently by default from the regular REST convention because pages are arranged in a tree, not a flat list. See below (copied from the code).

"Trees are arranged in a tree, not a list. So this API returns the home page,
with _children populated if ?_children=1 is in the query string. An editor can
also get a light version of the entire tree with ?all=1, for use in a
drag-and-drop UI.

If ?flat=1 is present, the pages are returned as a flat list rather than a tree,
and the `_children` property of each is just an array of `_id`s.

If ?autocomplete=x is present, then an autocomplete prefix search for pages
matching that string is carried out, and a flat list of pages is returned,
with no `_children`. This is mainly useful to our relationship editor.
The user must have some page editing privileges to use it. The 10 best
matches are returned as an object with a `results` property containing the
array of pages.

If querying for draft pages, you may add ?published=1 to attach a
`_publishedDoc` property to each draft that also exists in a published form."

Reply:
The main thing I'm flagging here is that it doesn't get "all pages" by default.